core/src/com/mygdx/insanerpg/Model/Unit/Unit.java

```diff
@@ -0,0 +1,15 @@
+package com.mygdx.insanerpg.Model.Unit;
+
+class Unit {
+    private int health;
```

Comment:
Are we having health as int? or double? also should have _

Reply:
I say int makes sense. Never seen a game used decimal healths